src/wp-admin/includes/image.php

```diff
@@ -482,6 +482,60 @@ function _wp_make_subsizes( $new_sizes, $file, $image_meta, $attachment_id ) {
 	return $image_meta;
 }
 
+/**
+ * Copy parent attachment properties to newly cropped image.
+ *
+ *
+ * @param string $cropped Path to the cropped image file.
```

Comment:
```suggestion
 * Copy parent attachment properties to newly cropped image.
 *
 * @since 6.5.0
 *
 * @param string $cropped Path to the cropped image file.
```

Add missing `@since`